source/runtime.hpp

```diff
@@ -82,6 +82,7 @@ namespace reshade
 		/// Captures a screenshot of the current back buffer resource and returns its image data in 32 bits-per-pixel RGBA format.
 		/// </summary>
 		bool capture_screenshot(uint8_t *pixels) final { return get_texture_data(get_back_buffer_resolved(get_current_back_buffer_index()), api::resource_usage::present, pixels); }
+		int execute_screenshot_postprocess(const std::filesystem::path &screenshot_path);
```

Comment:
I think it would make sense to move this into the `private:` section (and have it return `bool` instead of `int`, for consistency with other members of `runtime`).